src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -248,6 +228,53 @@ protected void handle(String consumeTag, Delivery delivery, T value, Confirmatio
         }
     }
 
+    private void subscribe() {
+        try {
+            consumerMonitor.updateAndGet(monitor -> {
```

Comment:
`updateAndGet` description states that update-function must be side-effect-free since it maybe re-applied multiple times due to thread contentio and the function we're providing has side-effects

Reply:
https://openliberty.io/blog/2021/09/21/building-on-atomic-reference.html

Reply:
[LazyInitializer](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/concurrent/LazyInitializer.html) also solves this problem

Reply:
Resolved in https://github.com/th2-net/th2-common-j/pull/244